sympy/series/limits.py

```diff
@@ -319,8 +319,14 @@ def remove_abs(expr):
 
         try:
             if str(dir) == '+-':
-                r = gruntz(e, z, z0, '+')
-                l = gruntz(e, z, z0, '-')
+                try:
+                    r = limit(e, z, z0, '+')
+                except (PoleError, ValueError):
+                    r = limit(e, z, z0, '+')
+                try:
+                    l = limit(e, z, z0, '-')
+                except (PoleError, ValueError):
+                    l = limit(e, z, z0, '-')
```

Comment:
A better fix would be

if str(dir) == '+-':
    r = limit(e, z, z0, '+')
    l = limit(e, z, z0, '-')

`limit` would take care of appropriate algorithm to be used for limit evaluation.

Reply:
This one is a really silly error. The first one should be `gruntz()`. `limit()` is supposed to be used only if `gruntz()` raises `ValueError` or `PoleError`.

Reply:
I would still suggest directly calling `limit()` rather than using `Gruntz` and catching error because it can hide bugs and `limit` would eventually call `Gruntz` with appropriate pre processing if necessary.

Reply:
Yeah right makes sense